internal/lightcone/abundance/timewaitsfornoone/timewaitsfornoone.go

```diff
@@ -0,0 +1,113 @@
+package timewaitsfornoone
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/lightcone"
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	time key.Modifier = "time-waits-for-no-one"
+	// key for extra dmg attack, not for modifier.
+	extraDmg key.Attack = "time-waits-for-no-one-extra-damage"
```

Comment:
nit: I would use the same key for both modifier and attack, since context will make it clear if the key is being used in the modifier or the attack. Would change this to:
```go
name = "time-waits-for-no-one"
```

This makes the `name` variable an `untyped string` which will work as both `key.Modifier` and `key.Attack`